setup.py

```diff
@@ -17,6 +17,7 @@
     'redis>=2.10',
     'requests>=2.2.0',
     'ruamel.yaml',
+    'importlib-metadata; python_version < "3.8"'
```

Comment:
Debian 10 buster has Python 3.7 and `python3-importlib-metadata` is only in buster-backports. So please update the installation instructions, adding that activating backports on Debian 10 is necessary and add the package as dependency to `debian/control`. I think it's totally fine to require backports on an old OS version.